src/api/merge/importWhitelist.service.ts

```
@@ -0,0 +1,139 @@
+
+        const address = row[0].replace(/"/g, "").toLowerCase();
+
+        // Validate Ethereum address
+        if (!isAddress(address)) {
+          this.logger.warn(`Invalid address at line ${i + 2}: ${address}`);
+          continue;
+        }
+
+        const amount = row[1].replace(/,/g, "").replace(/"/g, "");
+
+        if (!amount || isNaN(Number(amount))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)